### PR TITLE
Fixed: Added new and missing translations

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
title

#### Todos
- Tests
- [x] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 1]
```diff
@@ -130,6 +131,7 @@
   "ContinuingAllBooksDownloaded": "Continuing (All books downloaded)",
   "ContinuingMoreBooksAreExpected": "More books are expected",
   "ContinuingNoAdditionalBooksAreExpected": "No additional books are expected",
+  "ConvertToFormat": "ConvertToFormat",
```
Comment (ta264) [inline src/NzbDrone.Core/Localization/Core/en.json]: This needs fixing I think

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 2]
```diff
@@ -842,12 +864,12 @@
...
   "WriteBookTagsHelpTextWarning": "Selecting 'All files' will alter existing files when they are imported.",
+  "WriteMetadataTags": "Write Metadata Tags",
   "WriteTagsAll": "All files; initial import only",
   "WriteTagsNew": "For new downloads only",
   "WriteTagsNo": "Never",
   "WriteTagsSync": "All files; keep in sync with Goodreads",
   "Year": "Year",
   "YesCancel": "Yes, Cancel",
-  "Yesterday": "Yesterday",
-  "UpdateAvailable": "New update is available"
```
Comment (ta264) [inline src/NzbDrone.Core/Localization/Core/en.json]: Why is this removed?

**Review by ta264 — CHANGES_REQUESTED** [review]

File: src/NzbDrone.Core/Localization/Core/en.json (hunk 2 above)
Comment (bakerboy448) [reply src/NzbDrone.Core/Localization/Core/en.json]: It's not; just properly ordered alphabetically

File: src/NzbDrone.Core/Localization/Core/en.json (hunk 1 above)
Comment (bakerboy448) [inline src/NzbDrone.Core/Localization/Core/en.json]:
```suggestion
  "ConvertToFormat": "Convert To Format",
```

File: frontend/src/Bookshelf/BookshelfFooter.js [hunk 3]
```diff
@@ -114,7 +115,7 @@ class BookshelfFooter extends Component {
 
         <div className={styles.inputContainer}>
           <div className={styles.label}>
-            Monitor Existing Books
+            M{translate('MonitorExistingBooks')}
```
Comment (ta264) [inline frontend/src/Bookshelf/BookshelfFooter.js]: Needs fixing

**Review by ta264 — APPROVED** [review]
